rusk/src/lib/lib.rs

```diff
@@ -778,8 +813,22 @@ fn execute(
         )
         .expect("Refunding must succeed");
 
-    res.map(|res| res.map(|data| (data, gas_spent)))
-        .unwrap_or(Ok((vec![], gas_spent)))
+    res.map(|res| res.map(|receipt| (receipt, gas_spent)))
+        .unwrap_or(Ok((
+            CallReceipt {
+                points_spent: 0,
```

Comment:
wait... should this be like?
```suggestion
                points_spent: gas_spent,
```